AUTHORS

```diff
@@ -1266,4 +1268,50 @@ Augusto Borges <borges.augustoar@gmail.com>
 Han Wei Ang <ang.h.w@u.nus.edu>
 Pablo <48098178+PabloRuizCuevas@users.noreply.github.com>
 Congxu Yang <u7189828@anu.edu.au>
-Saicharan <62512681+saicharan2804@users.noreply.github.com>
\ No newline at end of file
+Saicharan <62512681+saicharan2804@users.noreply.github.com>
+Arnab Nandi <arnabnandi2002@gmail.com>
+Harrison Oates <48871176+HarrisonOates@users.noreply.github.com>
+Corey Cerovsek <corey@cerovsek.com>
+Harsh Kasat <hkasat@waymore.io>
+omahs <73983677+omahs@users.noreply.github.com>
+PascalGitz <pascal.gitz@hotmail.ch>
+Ravindu-Hirimuthugoda <ravindu.18@cse.mrt.ac.lk>
+Sophia Pustova <tripplezzed@gmail.com>
+George Pittock <gpittock4@gmail.com>
+Warren Jacinto <warrenjacinto@gmail.com>
+sachinSingh16-09 <sachinishu02@gmail.com>
+Zedmat <104870914+harshkasat@users.noreply.github.com>
+samithkavishke <55777141+samithkavishke@users.noreply.github.com>
+Viraj Vekaria <virajv5593@gmail.com>
+Ankit Kumar Singh <ankitdiswar10@gmail.com>
+Abhishek Kumar <abhishek.nitdelhi@gmail.com>
+Senku <mohakmalviya2000@gmail.com>
+Matthias Liesenfeld <116307294+maliesen@users.noreply.github.com>
+dodo <palumbododo@gmail.com>
+Mohamed Rezk <mohrizq895@gmail.com>
+T-vaccari <05-gesto-follemente@icloud.com>
+Dodopalu <87149525+Dodopalu@users.noreply.github.com>
```

Comment:
@Dodopalu you have two entries in the .mailmap file and so two entries in the AUTHORS file. This should be changed so that there is not a duplicate entry.

What name and email address do you want to record in the AUTHORS file.

Reply:
I've removed one duplicate entry.